docs/rewards/reward-boosts.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: reward-boosts
+title: Reward Boosts
+sidebar_label: Reward Boosts
+slug: /project/reward-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+### Overview
+
+**Reward boosts** are additional rewards given to stations that contribute weather data during daily distribution,in addition to the [**base rewards**](/reward-mechanism#daily-rewards-calculation-process). Learn more about the daily rewards in the [Reward Mechanism (RM) ](/reward-mechanism) section.
```

Comment:
daily distribution,in addition to the -----> daily distribution, in addition to the